rgw/v2/tests/s3_swift/test_multisite_sync_policy.py

```diff
@@ -0,0 +1,256 @@
+""" test_Mbuckets_with_Nobjects.py - Test with M buckets and N objects
+
+Usage: test_multisite_sync_policy.py -c <input_yaml>
+
+<input_yaml>
+	Note: Any one of these yamls can be used
+	test_multisite_sync_policy.yaml
+
+Operation:
+	Creates and delete sync policy group
+	Creates and delete sync policy flow
+    Creates and delete sync policy pipe
+"""
+# test basic creation of buckets with objects
+import os
+import sys
+from random import randint
+
+sys.path.append(os.path.abspath(os.path.join(__file__, "../../../..")))
+import argparse
+import json
+import logging
+import traceback
+
+import v2.lib.resource_op as s3lib
+import v2.utils.utils as utils
+from v2.lib.exceptions import RGWBaseException, TestExecError
+from v2.lib.resource_op import Config
+from v2.lib.rgw_config_opts import CephConfOp
+from v2.lib.s3.auth import Auth
+from v2.lib.s3.write_io_info import BasicIOInfoStructure, IOInfoInitialize
+from v2.tests.s3_swift import reusable
+from v2.utils.log import configure_logging
+from v2.utils.test_desc import AddTestInfo
+from v2.utils.utils import RGWService
+
+log = logging.getLogger()
+TEST_DATA_PATH = None
+
+
+def get_multisite_info():
```

Comment:
this method can be used by other also right , can we put it in reusables.py?

Reply:
Done